### PR TITLE
Fix incorrect references of default branch to development and...

### PR DESCRIPTION
**Pull request by lamdbui**
use text decoration to highlight branch names.

### CODE REVIEW

**Review by andino93 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,33 +2,34 @@
...
+`feature` -> `development`* -> `master` -> `staging` -> `deployment`
 
-- Deployment: actual production environment available publicly
-- Staging: simulated deployed state only available internally
-- Master: always "good" source of truth. Only certain members can merge into master
-- Develop: "experimental / latest and greatest" code and default branch for all members
-- Feature: individual feature implementations per-user
+\* default branch
+
+- `deployment`: actual production environment available publicly
```
Comment (mikehern) [inline CONTRIBUTING.md:9]: Should this be `development` and not `deployment`?

**Review by mikehern — CHANGES_REQUESTED** [review]
1 typo.  Can approve and merge after fix.

File: CONTRIBUTING.md (hunk 1 above)
Comment (andino93) [reply CONTRIBUTING.md:9]: `deployment` is correct. It refers to the deployed and working version of our app. `development` is where we're building features from and working on bleeding edge code.

Comment (mikehern) [reply CONTRIBUTING.md:9]: Got it! Deployment is the equivalent of production.

**Review by mikehern — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (lamdbui) [reply CONTRIBUTING.md:9]: yes, @andino93 is correct. Did you guys want to change it to `production` instead? I think I just had it as `deployment` as a habit.

Comment (mikehern) [reply CONTRIBUTING.md:9]: It's fine-- I got confused on the semantics but we're good from here.  Thank you both.

**Pull request merged**